tools/metadata/main_test.go

```diff
@@ -65,7 +67,7 @@ GET /undocumented/{undocumented_id}
 `, "")
 }
 
-func TestUpdateOpenAPI(t *testing.T) {
+func TestUpdateOpenAPI(t *testing.T) { //nolint:paralleltest
 	// t.Parallel() - cannot use when helper calls t.Setenv
```

Comment:
`nolint` support comments, like this:

```suggestion
//nolint:tparallel,paralleltest // cannot use t.Parallel() when helper calls t.Setenv
func TestUpdateOpenAPI(t *testing.T) {
```